lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -154,7 +155,7 @@
 <hr class="divider" />
 %endif
 
-<div class="member-lists-management membership-section">
+<div class="member-lists-management membership-section" aria-live="polite">
```

Comment:
same as above.  If you only plan on injecting an error message in here, then its fine.  In fact, it's great :)